website/landingpage/dev/package.json

```diff
@@ -17,26 +17,26 @@
     "what-input": "^5.2.12"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/preset-env": "^7.9.6",
-    "@babel/register": "^7.9.0",
-    "autoprefixer": "^9.8.0",
-    "babel-loader": "^8.1.0",
-    "browser-sync": "^2.28.1",
+    "@babel/core": "7.23.6",
```

Comment:
This is now a fixed version. Is that intentional?

Reply:
not intentional, changed 👍